onward/app/controllers/TechFeedbackController.scala

```diff
@@ -1,11 +1,53 @@
 package controllers
 
+import javax.inject.Inject
+
+import conf.Configuration
 import common._
 import model.Cached.RevalidatableResult
 import model.{ApplicationContext, Cached, MetaData, SectionSummary}
+import play.api.data.Form
+import play.api.libs.json.Json
 import play.api.mvc.{Action, Controller}
+import play.api.libs.ws._
+import play.api.data.Forms._
+import scala.concurrent.duration._
+
+class TechFeedbackController(ws: WSClient) (implicit context: ApplicationContext) extends Controller with Logging {
+
+  def submitFeedback(path: String) = Action { implicit request =>
+
+    val feedbackForm = Form(
+      tuple(
+        "category" -> text,
+        "body" -> text,
+        "user" -> text,
+        "extra" -> text,
+        "name" -> text
+      )
+    )
```

Comment:
Working with a custom data type (ie: `case class TechFeedbackData(category...)`) might be a bit easier than using a tuple. For example you could get the Json Writes for free. 
Not a big deal because this controller is quite simple though, so feel free to ignore this comment :D